code/controllers/configuration/entries/general.dm

```diff
@@ -638,6 +638,15 @@
 
 /datum/config_entry/flag/picture_logging_camera
 
+/datum/config_entry/number/roundstart_logout_report_time_average
+	default = 10 MINUTES
+	integer = FALSE
```

Comment:
in what scenario will you want these to support FPNs?

Reply:
Honestly, not really?